### PR TITLE
(#71) Expanded the social media section and fixed the icon bug.

### PR DESCRIPTION
**Pull request by AppleC4T**
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes https://github.com/fossasia/gci18.fossasia.org/issues/84
- Preview Link: https://applec4t.github.io/gci18.fossasia.Fork/
Before:
image
Before with the reverted PR (https://github.com/fossasia/gci18.fossasia.org/pull/79)
image
After:
image

## Description / Changes
Expanded the social media section with new links and added a new CSS file for creating new styles instead of writing it in the deafult file, which causes bugs. For more info about the bugs check the reverted PR:  https://github.com/fossasia/gci18.fossasia.org/pull/79 .

### CODE REVIEW

**Comment by saarthakchats** [comment]
This is no longer the case #82 was reverted

**Comment by AppleC4T** [comment]
@saarthakchats 
> This is no longer the case #82 was reverted

It also reverted the changes I made. This PR brings them back and fixes the bug within them.

**Comment by STEdirisuriya** [comment]
LGTM!

File: index.html [hunk 1]
```diff
@@ -140,10 +140,15 @@ <h3> Students </h3>
         <div class="row mb80 mb-xs-0">
             <div class="col-md-8 col-md-offset-2 text-center">
                 <h1>Connect with Fossasia on Social Media</h1>
+				<p>Be sure to follow us on our following social media!</p>
                 <p class="large">
-                    <a href="https://www.facebook.com/fossasia/" class="ti-facebook" target="_blank"></a>
-                    <a href="https://twitter.com/fossasia" class="ti-twitter" target="_blank"></a>
-
+<a href="https://www.facebook.com/fossasia/" class="ti-facebook" target="_blank"></a>
```
Comment (tstreamDOTh) [inline index.html]: Indent properly, rest looks great

**Comment by AppleC4T** [comment]
@tstreamDOTh Check now, should be ok.

**Comment by realslimshanky** [comment]
@AppleC4T I wouldn't recommend solving two different issues with a single pull request.

**Comment by AppleC4T** [comment]
@realslimshanky the issue #71 is currently closed, because as I said the previous changes were reverted. This PR adds them again but ensures the bug doesn't occur again, so it basically solves one issue. (look: https://github.com/fossasia/gci18.fossasia.org/pull/79)

**Comment by realslimshanky** [comment]
@AppleC4T why are you adding the same changes to this PR when they have already been merged in the master branch. Just rebase your branch to pull the latest changes and work only what is needed.

**Comment by AppleC4T** [comment]
@realslimshanky There were merged, but then reverted because of a bug in the previous PR, so they are currently not in the master.

**Comment by AppleC4T** [comment]
@realslimshanky the description should be clearer now.

File: _includes/header.html [hunk 2]
```diff
@@ -14,6 +14,7 @@
   <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
   <link href='http://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600'
     rel='stylesheet' type='text/css'>
+	<link href="css/socialmedia-icons.css" type="text/css" rel="stylesheet" />
```
Comment (fragm3) [inline _includes/header.html]: Indent this line properly

**Comment by AppleC4T** [comment]
@fragm3 fixed the indent.

**Comment by fragm3** [comment]
@AppleC4T Fix indentation for `index.html` also, for the changes you did.

**Comment by AppleC4T** [comment]
@fragm3 is it ok now?

**Pull request closed without merging**